libs/custom-paper-handler/src/driver/driver.ts

```diff
@@ -587,7 +683,7 @@ export class PaperHandlerDriver {
     const unsignedNumMotionUnits: Uint16 =
       numMotionUnits < 0 ? Uint16Max + 1 - numMotionUnits : numMotionUnits;
     const [nH, nL] = Uint16toUint8(unsignedNumMotionUnits);
-    return this.transferOutGeneric([...SET_RELATIVE_PRINT_POSITION, nL, nH]);
+    return this.transferOutGeneric(SetRelativePrintPositionCommand, { nL, nH });
```

Comment:
Earlier in this function and elsewhere we hardcode -32768 and 32867. I think we should create constants for `Int16Min` and `Int16Max`.

Reply:
https://github.com/votingworks/vxsuite/pull/3545/commits/a4ee5c49e620f137155fcdd8a7b1326667793ca7